frontend/public/components/notification-drawer.tsx

```diff
@@ -288,7 +312,8 @@ export const ConnectedNotificationDrawer_: React.FC<ConnectedNotificationDrawerP
     <NotificationDrawer
       className="co-notification-drawer"
       isInline={isDesktop}
-      isExpanded={isDrawerExpanded}
+      isExpanded={isDrawerExpanded && !isDrawerAnimating}
+      isDrawerHidden={isDrawerHidden}
```

Comment:
I'm confused... There are separate `isExpanded` and `isDrawerHidden` props? What is the difference between those?

Reply:
@spadgett Yea, so this will go away when the actual component fix is completed, but the idea is that the panel component is set to hidden when it finishes animation so the focus doesn't expose the closed panel. When the drawer is fixed, this can be removed.